src/pkg/control/options.go

```diff
@@ -105,4 +105,7 @@ type Toggles struct {
 	ExchangeImmutableIDs bool `json:"exchangeImmutableIDs,omitempty"`
 
 	RunMigrations bool `json:"runMigrations"`
+
+	// DisableConcurrencyLimiter disables the concurrency limiter package
```

Comment:
Nit, since you're not technically turning off a package.
```suggestion
	// DisableConcurrencyLimiter removes concurrency restraints when communicating with graph API
```

Reply:
Thanks!